source/people/main.rst

```diff
@@ -107,6 +107,11 @@ Undergraduate Students & Interns
 |                                    |                                    |                                     |
 | :doc:`Piper Lincoln <plincoln>`    | :doc:`Matthew Nyberg <mnyberg>`    | :doc:`Brad Laufenberg <laufenberg>` |
 +------------------------------------+------------------------------------+-------------------------------------+
++------------------------------------+------------------------------------+-------------------------------------+
+|                                    |                                    |                                     |
+|                                    |                                    |                                     |
+| :doc:`Yuehan Qin <yuehan>`         |                                    |                                     |
++------------------------------------+------------------------------------+-------------------------------------+
```

Comment:
Rather than have a new row, you can add yourself as a fourth entry on the same row as the others.